client/src/amplify-ui/cli-feature-flag/feature-flags.json

```diff
@@ -356,6 +356,27 @@
             "defaultExistingProject": true
           }
         ]
+      },
+      "enableDartNullSafety": {
+        "description": "Generate Dart models with null safety for Flutter applications using DataStore. Refer to https://docs.amplify.aws/lib/project-setup/null-safety/q/platform/flutter for more information",
```

Comment:
I think that we usually try to avoid hard coding URLs that are internal to the docs site? Instead, I think we generally [use this pattern](https://github.com/aws-amplify/docs/blob/281b0dbe1a213606a3a00543bbc7d8240c2ec509/docs/lib/project-setup/fragments/native_common/prereq/flutter_null_safety.md).

Reply:
Markdown or hyperlinks are not supported here as it's a json file that stores the Feature Flag metadata.

Reply:
Ha - sorry, I totally missed that obvious fact and focused on the text. I'm approving.